Dockerfile

```diff
@@ -21,8 +26,8 @@ RUN apt-get update -qq && apt-get install --no-install-recommends -y build-essen
         python3-sqlalchemy python3-asyncpg \
         python3-icu python3-datrie python3-yaml python3-jinja2 \
         python3-pip \
-        && apt-get clean \
-        && rm -rf /var/lib/apt/lists/
+    && DEBIAN_FRONTEND=noninteractive apt-get clean \
+    && rm -rf /var/lib/apt/lists/
```

Comment:
With the cache it should be removed :-)

Reply:
not sure, as the cache is caching `/var/cache` and not `/var/lib/apt/lists/`

Reply:
but I agree that the `apt-get clean` can possibly be removed